src/build.cc

```diff
@@ -135,8 +135,13 @@ void BuildStatus::BuildEdgeFinished(Edge* edge,
   if (!success) {
     string outputs;
     for (vector<Node*>::const_iterator o = edge->outputs_.begin();
-         o != edge->outputs_.end(); ++o)
-      outputs += (*o)->path() + " ";
+         o != edge->outputs_.end(); ++o) {
+      if((*o)->path().find(' ') != string::npos) {
+        outputs += "'" + (*o)->path() + "' ";
```

Comment:
' isn't Windows-friendly.

(Is it worth worrying about other things like \s being in there if you're planning on parsing in a script?)

Reply:
Regarding 1: Using double quotes you'll also have to escape $.

I think we should add `'` (and escape all `'` in the path) on anything non-Windows.